package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gatling-enterprise-action",
+  "description": "Gatling Enterprise GitHub Action",
+  "repository": {
+    "type": "git",
+    "url": "git+https://gatling/enterprise-action.git"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/gatling/enterprise-action/issues"
```

Comment:
@slandelle Don't we want to centralize on `gatling/gatling/issues`? perhaps with a new `module:action` label?

Reply:
Changed to `gatling/gatling/issue`